packages/driver/cypress/integration/e2e/multi-domain/multi_domain_yield_spec.ts

```diff
@@ -97,41 +95,41 @@ describe('multi-domain yields', { experimentalSessionSupport: true, experimental
         symbol: Symbol(''),
       }
     }).then((obj) => {
-      // This will fail accessing the symbol
       // @ts-ignore
-      return obj.symbol
+      return obj.symbol // This will fail accessing the symbol
     })
   })
 
   it('succeeds if subject cannot be serialized and is not accessed', () => {
     cy.switchToDomain('foobar.com', () => {
-      cy
-      .get('[data-cy="dom-check"]')
-    }).then((obj) => {
+      cy.get('[data-cy="dom-check"]')
+    })
+    .then(() => {
       return 'object not accessed'
-    }).should('equal', 'object not accessed')
+    })
+    .should('equal', 'object not accessed')
   })
 
   it('throws if subject cannot be serialized and is accessed', (done) => {
     cy.on('fail', (err) => {
-      assertLogLength(logs, 8)
-      expect(logs[7].get('error')).to.eq(err)
+      assertLogLength(logs, 6)
+      expect(logs[5].get('error')).to.eq(err)
       expect(err.message).to.include('`cy.switchToDomain()` could not serialize the subject due to one of its properties not being supported by the structured clone algorithm.')
 
       done()
     })
 
     cy.switchToDomain('foobar.com', () => {
-      cy
-      .get('[data-cy="dom-check"]')
-    }).invoke('text')
+      cy.get('[data-cy="dom-check"]')
+    })
+    .then((subject) => subject.text())
```

Comment:
This fails quicker than `invoke('text')` because `invoke` will retry until the timeout (4000ms) whereas `then` does not retry.